webapp/app/templates/basis/footer.html

```diff
@@ -4,61 +4,25 @@
 {% else %}
     {% set disable_extended_footer = true %}
 {% endif %}
-<footer class="pl-0 pr-0 ml-0 mr-0 mt-0 {{'border-0' if disable_extended_footer }}">
-    {% if not disable_extended_footer %}
-    <div class="container-fluid pt-4 pb-3 pl-0 pr-0">
-        <div class="footer-inner">
-            <div class="row m-0">
-                <div class="col-lg-3 col-md-4 col-sm-6 pb-4">
-                    <span class="bold">{{ _('footer.help-contact') }}</span>
-                    <ul class="list-unstyled">
-                        <li class="text-nowrap mt-3">
-                            <a href="{{ url_for('howitworks') }}">{{ _('footer.information') }}</a>
-                        </li>
-                        <li class="text-nowrap mt-2">
-                            <a href="{{ url_for('contact') }}">{{ _('footer.contact') }}</a>
-                        </li>
-                        <li class="text-nowrap mt-2">
-                            <a href="{{ url_for('relatives_info') }}">{{ _('footer.relatives') }}</a>
-                        </li>
-                        <li class="text-nowrap mt-2">
-                            {% if current_user.is_authenticated %}
-                            <span class="inactive">{{ _('footer.revocation') }}</span>
-                            {% else %}
-                            <a href="{{ url_for('unlock_code_revocation', step='start') }}">{{ _('footer.revocation')}}</a>
-                            {% endif %}
-                        </li>
-                    </ul>
-                </div>
-                <div class="col-lg-3 col-md-4 col-sm-6 pb-4">
-                    <span class="bold">{{ _('footer.about-product') }}</span>
-                    <ul class="list-unstyled">
-                        <li class="text-nowrap mt-3">
-                            <a href="{{ url_for('about_steuerlotse') }}">{{ _('footer.about-steuerlotse') }}</a>
-                        </li>
-                        <li class="text-nowrap mt-2">
-                            <a href="{{ url_for('about_digitalservice') }}">{{ _('footer.about-ds') }}</a>
-                        </li>
-                    </ul>
+<footer class="pl-0 pr-0 ml-0 mr-0 mt-0">
```

Comment:
You removed disable_extended_footer but we still using it in the backend:
![image](https://user-images.githubusercontent.com/91742583/177108736-af3a1ef3-f389-4176-97ba-b295b49e56b6.png)

Can you delete all references?